### DISCUSSION TITLE
How do I set mint price and max supply

### DISCUSSION THREAD

**Discussion by hamood1337** [TENK-DAO/tenk#34] [Q&A]
Coming from Solidity, Rust looks confusing. 
I can't seem to find where the contract sets the mint price and the max supply.

Would love any pointers on how to change it.

**Comment by willemneal** [comment]
It's set in the init function `new_default_meta` 

Here are the TS types:
https://github.com/TENK-DAO/tenk/pull/33/files#diff-ddbeda77499421023bbeef28d10392c263a968188b0cb54d5c128e11b5f0a573R206

Here is the corresponding rust code: https://github.com/TENK-DAO/tenk/blob/325d570f84e7795e892b86904d61d7ae692f9454/contracts/tenk/src/lib.rs#L187

So it's `size` and `PriceStructure`.
